src/pages/authorization/authorization-in-practice/index.mdx

```diff
@@ -139,22 +148,22 @@ If you're struggling with errors, check out [Authorizaton FAQ](/getting-started/
 
 Managed to get your token? Great! You can now use it to authorize the request sent the **List Chats** request.
 
-Go to <a href="https://developers.livechat.com/docs/messaging/agent-chat-api/v3.2/#list-chats" target="_blank">List Chats</a> in our documentation and copy the request code snippet. Once more, we'll use Postman and its **Import > Paste Raw Text** option. Replace _<your_access-token>_ with the actual token and click **Import**.
+Go to <a href="https://developers.livechat.com/docs/messaging/agent-chat-api/v3.2/#list-chats" target="_blank">List Chats</a> in our documentation and copy the request code snippet. Once more, we'll use Postman and its **Import > Paste Raw Text** option. Replace _<your_access-token>_ with the actual token and click **Continue**. When the following window appears on your screen, click **Import**.
```

Comment:
```suggestion
Go to <a href="https://developers.livechat.com/docs/messaging/agent-chat-api/v3.4/#list-chats" target="_blank">List Chats</a> in our documentation and copy the request code snippet. Once more, we'll use Postman and its **Import > Paste Raw Text** option. Replace _<your_access-token>_ with the actual token and click **Continue**. When the following window appears on your screen, click **Import**.
```

Reply:
It would be better to showcase v3.4 as it's the stable version. You might need to retake a screenshot or two :<